src/components/DetailsMetrics/detailsMetrics.utils.js

```diff
@@ -19,10 +19,35 @@ such restriction.
 */
 import { capitalize, chain } from 'lodash'
 
-const mlrunInfra = 'mlrun-infra'
 const metricsColorsByFullName = {}
 const usedColors = new Set()
 
+export const timeRangeMapping = {
+  past24hours: 'last 24 day',
+  pastWeek: 'last week',
+  pastMonth: 'last month',
+  pastHour: 'last hour',
+  customRange: 'custom range'
+}
```

Comment:
don't use hardcoded ids of the date picker, use constants

Reply:
done.